HW_3/task_2.py

```diff
@@ -0,0 +1,44 @@
+# слов и вернуть 10 самых частых. Не учитывать знаки препинания
+# и регистр символов. За основу возьмите любую статью
+# из википедии или из документации к языку.
+
+str1 = "Массив — структура данных, хранящая набор значений (элементов массива), " + \
+    "идентифицируемых по индексу или набору индексов, принимающих целые (или приводимые к целым) " + \
+    "значения из некоторого заданного непрерывного диапазона. Одномерный массив можно рассматривать " + \
+    "как реализацию абстрактного типа данных — вектор. В некоторых языках программирования массив может " + \
+    "называться также таблица, ряд, вектор, матрица. Размерность массива — это количество индексов, " + \
+    "необходимое для однозначной адресации элемента в рамках массива. По количеству используемых индексов " + \
+    "массивы делятся на одномерные, двумерные, трёхмерные и т. д. Форма или структура массива — сведения о количестве " + \
+    "размерностей и размере (протяжённости) массива по каждой из размерностей; может быть представлена одномерным массивом. " + \
+    "Особенностью массива как структуры данных (в отличие, например, от связного списка) является " + \
+    "константная вычислительная сложность доступа к элементу массива по индексу. Массив относится к структурам данных " + \
+    "с произвольным доступом. В простейшем случае массив имеет константную длину по всем размерностям и может хранить данные " + \
+    "только одного, заданного при описании, типа. Ряд языков поддерживает также динамические массивы, " + \
+    "длина которых может изменяться во время выполнения программы, и гетерогенные массивы, " + \
+    "которые могут в разных элементах хранить данные различных типов. Некоторые специфичные типы массивов, " + \
+    "используемые в различных языках и реализациях — ассоциативный массив, дерево отрезков, V-список, параллельный массив, " + \
+    "разреженный массив. Основные достоинства использования массивов — лёгкость вычисления адреса элемента по его индексу " + \
+    "(поскольку элементы массива располагаются один за другим), одинаковое время доступа ко всем элементам, " + \
+    "малый размер элементов (они состоят только из информационного поля). Среди недостатков — невозможность удаления " + \
+    "или добавления элемента без сдвига других при использовании статических массивов, а при использовании динамических " + \
+    "и гетерогенных массивов — более низкое быстродействие из-за накладных расходов на поддержку динамики и разнородности. " + \
+    "При работе с массивами с реализацией по типу языка Си (с указателями) и отсутствии дополнительных средств контроля " + \
+    "типичной ошибкой времени выполнения является угроза выхода за границы массива и повреждения данных."
+
+
+str2 = str1.lower().replace('.', '').replace(',', '').replace(
+    '(', '').replace(')', '').replace('—', '').replace(';', '')
+list1 = str2.split()
+my_dict = {}
+
+for item in list1:
+    my_dict.setdefault(item, 0)
+    my_dict[item] = my_dict[item] + 1
```

Comment:
Использование метода `get` было бы проще.
```suggestion
    my_dict[item] = my_dict.get(item, 0) + 1
```